### PR TITLE
OLS-598: Script to download object from Ceph

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Script to download object from Ceph
+Documentation

## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #[OLS-598](https://issues.redhat.com//browse/OLS-598)
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: scripts/data_collection/README.md [hunk 1]
```diff
@@ -60,3 +60,31 @@ Run:
...
+Run:
+
+```
+python download_object.py key
+```
+
+An example:
+
+```
+python download_object.py archives/compressed/b6/b6cf044a-4870-4a8c-a849-1b0016cc8171/202404/25/185345.tar.gz
```
Comment (onmete) [inline scripts/data_collection/README.md:89]: rename to `download_ols_archive.py`?:)

Comment (tisnik) [reply scripts/data_collection/README.md:89]: that's actually good name, today I'm now good with naming things :) changing

Comment (tisnik) [reply scripts/data_collection/README.md]: fixed @onmete

**Pull request merged**